themes/src/main/resources/theme/keycloak/welcome/index.ftl

```diff
@@ -22,7 +22,9 @@
     <div class="pf-v5-c-login">
       <div class="pf-v5-c-login__container">
         <header class="pf-v5-c-login__header">
-          <img class="pf-v5-c-brand" src="${resourcesPath}/logo.svg" alt="${productName} Logo">
+          <div class="pf-v5-c-brand">
+            <img src="${resourcesPath}/logo.svg" alt="${productName} Logo" style="height: 63px;width: 300px;display: block;margin: 0 auto;">
```

Comment:
Can we not use in-line styles for this? There is already a stylesheet present for the welcome page that can be used.